libs/core/src/lib/services/usb.serial.service.ts

```diff
@@ -8,10 +8,17 @@ export class UsbSerialService {
     protected static instance;
 
     /**
-     * Get instance
+     * @Deprecated - Get instance,  Use the new {@link createInstance} instead.
      * @returns
      */
     public static getInstance() {
+    }
```

Comment:
The `getInstance` function will returns the value of the new `createInstance` function. Maybe we should add the call here to be more explicit for any dev user.